code/modules/research/fabricators.dm

```diff
@@ -297,6 +297,7 @@
 			*/
 			being_built.forceMove(L) //Put the thing in the lockbox
 			L.name += " ([being_built.name])"
+			L.fits_max_w_class = 0 //You can't put things back into these boxes to prevent breaking arbitrary items
```

Comment:
That made more sense when I was trying to have more generalized failure effects for like all obj/weapon/gun/energy, and didn't want someone putting the cap's laser or something in a lockbox and breaking it